web/server/codechecker_server/api/mass_store_run.py

```diff
@@ -991,11 +991,11 @@ def __store_reports(
         """ Parse up and store the plist report files. """
         def get_skip_handler(
             report_dir: str
-        ) -> Optional[skiplist_handler.SkipListHandler]:
+        ) -> skiplist_handler.SkipListHandler:
```

Comment:
If something goes wrong in the code below and an exception is thrown we will return with `None` from this function.

Reply:
I modified the code. Exception will raise when any error occurred XOR an usable SkipListHandler instance returned.
The server survive exceptions raised from here (or from somewhere else), because it handles exceptions in store() function at line 1105.
Client get API error in that case.